tensorflow/lite/micro/micro_allocator.cc

```diff
@@ -880,7 +887,7 @@ TfLiteStatus MicroAllocator::CommitStaticMemoryPlan(
 
   const SubGraph* subgraph = model->subgraphs()->Get(subgraph_idx);
   size_t allocation_info_count =
-      subgraph->tensors()->size() + scratch_buffer_request_count_;
+      subgraph->tensors()->size() + total_scratch_requests_;
```

Comment:
subgraph->tensors()->size() + subgraph_allocations[subgraph_idx].scratch_buffer_request_count_ should work, right?

Reply:
Good call. I've added it as an argument; subgraph_allocations is not accessible from within this method.